arai60/06-07_Stack/06_20_Valid Parentheses/level_2.py

```diff
@@ -0,0 +1,14 @@
+class Solution:
+    def isValid(self, s: str) -> bool:
+        symbols = ["dummy"]
+        mapping = {")": "(", "]": "[", "}": "{"}
+        for i_char in s:
+            if i_char == "(" or i_char == "[" or i_char == "{":
```

Comment:
せっかくL4でmapを作ってるのでキーの存在有無で確認する方がシンプルかと。
```suggestion
            if i_char in mapping:
```